streams/es.m3u

```diff
@@ -87,14 +93,22 @@ http://217.125.136.93:8080/canalsierradecadiz576.m3u8
 https://cdnlive.codev8.net/rtvalive/smil:channel1.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalSurAndalucia.es" status="timeout",Canal Sur Andalucía (720p)
 https://cdnlive.shooowit.net/rtvalive/smil:channel1.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="CanalTaronja.es" status="online",Canal Taronja Osona i Moianés (1080p) [Not 24/7]
+https://ingest1-video.streaming-pro.com/videostream/ctvic/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalTaronja.es" status="blocked",Canal Taronja Osona i Moianés (480p) [Not 24/7]
 https://taronjavic.streaming-pro.com/hls/vic.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkSpain.es" status="online",Cartoon Network Spain (720p)
+https://playout.cdn.cartoonnetwork.com.br/playout_02/playlist.m3u8
```

Comment:
[already in br.m3u](https://github.com/iptv-org/iptv/blob/639e9c5a9d75ac84099c0e04c3ad9b6f978b94e1/streams/br.m3u#L49-L50)